### PR TITLE
Docblocks cleanup

### PR DESCRIPTION
**Pull request by mkruk-u2**

### CODE REVIEW

File: DataSource/DataSourceBuilderInterface.php [hunk 1]
```diff
@@ -6,18 +6,64 @@
 
 interface DataSourceBuilderInterface
 {
+    /**
```
Comment (mkruk-u2) [inline DataSource/DataSourceBuilderInterface.php]: All these methods where docblock-ed to get rid of _"Method 'addField' not found in..."_ IDE warning in DataSourceConfiguration classes.

File: DataSource/Driver/Doctrine/QueryBuilder/Filterer.php [hunk 2]
```diff
@@ -147,74 +145,78 @@ protected function buildCondition(FilterCondition $filterCondition, $token, Quer
...
-            return '1=2';
-        } elseif ($filterMethod == FilterCondition::METHOD_IS_NULL) {
-            if ($filterCondition->getValue()) {
-                return $queryBuilder->expr()->isNull($identifier);
-            }
-
-            return $queryBuilder->expr()->isNotNull($identifier);
-        } else {
-            throw new \Exception("Unknown filtering method \"$filterMethod\" for column \"" . $filterCondition->getFieldName() . '"');
+        switch ($filterMethod) {
```
Comment (mkruk-u2) [inline DataSource/Driver/Doctrine/QueryBuilder/Filterer.php]: - changes `if ... elseif ... elseif ... elseif ...` to switch/case (for me it looks simpler, more readable like this)
- removes duplicated condition check for `METHOD_NUMERIC_EQ` and `METHOD_DATETIME_EQ` (previously done twice, at lines 152 and 162)

File: DataSource/DataSourceBuilder.php [hunk 3]
```diff
@@ -52,9 +52,10 @@ class DataSourceBuilder implements DataSourceBuilderInterface
     private $dataSourceFactory;
 
     /**
-     * @param                          $entityClass
-     * @param EventDispatcherInterface $eventDispatcher
-     * @param ChoicesBuilder           $choicesBuilder
+     * @param string                     $entityClass
+     * @param EventDispatcherInterface   $eventDispatcher
+     * @param DataSourceFactoryInterface $dataSourceFactory
```
Comment (yakobe) [inline DataSource/DataSourceBuilder.php]: could break BC.

File: DataSource/DataSourceBuilderInterface.php (hunk 1 above)
Comment (yakobe) [inline DataSource/DataSourceBuilderInterface.php]: :+1:

**Pull request merged**